__tests__/app.test.js

```diff
@@ -378,6 +378,36 @@ describe('/api/articles/:id', () => {
             .expect(404)
             .then(({body}) => {
 
+                expect(body.message).toBe('404: Not found');
+            })
+        })
+    })
+})
+
+describe('/api/comments/:id', () => {
+    describe('STATUS 204', () => {
+        test('204: DELETE /api/comments/:id - deletes the comment with the given id and responds with no content', () => {
+            return request(app)
+            .delete('/api/comments/1')
+            .expect(204)
+        })
+    })
+
+    describe('ERRORS', () => {
+        test('404 returns \'404: Not found\' for route that does not exist', () => {
+            return request(app)
+            .delete('/api/j/1')
+            .expect(404)
+            .then(({body}) => {
+                expect(body.message).toBe('404: Not found');
+            })
+        })
+
+        test('404: GET /api/articles/:id - responds with \'404: Not found\' when passed an id that does not exist', () => {
+            return request(app)
+            .get('/api/comments/9999')
```

Comment:
Be careful when writing these error handlers that we are ensuring they fail first, there is no code in your controller/model for a delete to handle this error, this test is passing as you've done a .get here not a .delete.